sae_lens/config.py

```diff
@@ -179,6 +181,8 @@ class LanguageModelSAERunnerConfig(Generic[T_TRAINING_SAE_CONFIG]):
     training_tokens: int = 2_000_000
     store_batch_size_prompts: int = 32
     seqpos_slice: tuple[int | None, ...] = (None,)
+    disable_concat_sequences: bool = False
+    exclude_bos_between_sequences: bool = False
```

Comment:
Does it make sense to have the same options both here and in `pretokenize_runner`? I could see an argument that the `pretokenize_runner` arguments are too confusing / complicated if it's best to stick with just `bos` here, but can also see it being nice if we have the same options and thus same capabilities regardless of pretokenizing or not

Reply:
Yeah I was thinking the latter, it is sort of strange these have different options, but can see the argument that maybe the pretokenize runner version is too complex if we want to keep them separate. It looks like the 6.0 release will have a lot of breaking changes so if we're going to unify these now is probably a good time to do it.